kubernetes/Makefile

```diff
@@ -31,6 +37,12 @@ build/concent-secrets/%: $(CONCENT_SECRET_DIR)/%
 	mkdir --parents $(dir $@)
 	cp --archive "$<" $(dir $@)
 
+
+# Secret file with template
```

Comment:
This rule is no used now that the `db-secrets.yml` has been moved into the repository.